lib/xapi/test_file.rb

```diff
@@ -9,7 +9,7 @@ def initialize(attributes)
     end
```

Comment:
This should probably be called `class TestFiles` now, and live in `lib/xapi/test_files.rb`.